tiles/src/main/java/com/protomaps/basemap/feature/RegionInfos.java

```diff
@@ -7,107 +7,961 @@
 /**
  * Query hardcoded information about sub-national regions.
  * <p>
- * Embedded hand-curated data on sub-national regions of significant extents, to assist in labeling. Includes US and
- * Australian states, and Canadian provinces.
+ * Embedded hand-curated data on sub-national regions of significant extents, to assist in labeling.
+ * Includes US states, AU states and territories, CA provinces and territories, and other significant regions globally
  * </p>
  */
 public class RegionInfos {
 
   private static String data = """
-    Alabama|US|3.5|7.5
-    Alaska|US|3.5|7.5
-    Arizona|US|3.5|7.5
-    Arkansas|US|3.5|7.5
-    California|US|3.5|7.5
-    Colorado|US|3.5|7.5
-    Connecticut|US|3.5|7.5
-    Delaware|US|3.5|7.5
-    Florida|US|3.5|7.5
-    Georgia|US|3.5|7.5
-    Hawaii|US|3.5|7.5
-    Idaho|US|3.5|7.5
-    Illinois|US|3.5|7.5
-    Indiana|US|3.5|7.5
-    Iowa|US|3.5|7.5
-    Kansas|US|3.5|7.5
-    Kentucky|US|3.5|7.5
-    Louisiana|US|3.5|7.5
-    Maine|US|3.5|7.5
-    Maryland|US|3.5|7.5
-    Massachusetts|US|3.5|7.5
-    Michigan|US|3.5|7.5
-    Minnesota|US|3.5|7.5
-    Mississippi|US|3.5|7.5
-    Missouri|US|3.5|7.5
-    Montana|US|3.5|7.5
-    Nebraska|US|3.5|7.5
-    Nevada|US|3.5|7.5
-    New Hampshire|US|3.5|7.5
-    New Jersey|US|3.5|7.5
-    New Mexico|US|3.5|7.5
-    New York|US|3.5|7.5
-    North Carolina|US|3.5|7.5
-    North Dakota|US|3.5|7.5
-    Ohio|US|3.5|7.5
-    Oklahoma|US|3.5|7.5
-    Oregon|US|3.5|7.5
-    Pennsylvania|US|3.5|7.5
-    Rhode Island|US|3.5|7.5
-    South Carolina|US|3.5|7.5
-    South Dakota|US|3.5|7.5
-    Tennessee|US|3.5|7.5
-    Texas|US|3.5|7.5
-    Utah|US|3.5|7.5
-    Vermont|US|3.5|7.5
-    Virginia|US|3.5|7.5
-    Washington|US|3.5|7.5
-    West Virginia|US|3.5|7.5
-    Wisconsin|US|3.5|7.5
-    Wyoming|US|3.5|7.5
-    Alberta|CA|3.5|7.5
-    British Columbia|CA|3.5|7.5
-    Manitoba|CA|3.5|7.5
-    New Brunswick|CA|3.5|7.5
-    Newfoundland and Labrador|CA|3.5|7.5
-    Northwest Territories|CA|3.5|7.5
-    Nova Scotia|CA|3.5|7.5
-    Nunavut|CA|3.5|7.5
-    Ontario|CA|3.5|7.5
-    Prince Edward Island|CA|3.5|7.5
-    Quebec|CA|3.5|7.5
-    Saskatchewan|CA|3.5|7.5
-    Yukon|CA|3.5|7.5
-    New South Wales|AU|4.6|8.1
-    Queensland|AU|4.6|8.1
-    South Australia|AU|4.6|8.1
-    Tasmania|AU|4.6|8.1
-    Victoria|AU|4.6|8.1
-    Western Australia|AU|4.6|8.1
-    Australian Capital Territory|AU|4.6|8.1
-    Northern Territory|AU|4.6|8.1
+    Australian Capital Territory|AU-ACT|3.5|7.5|Q3258
```

Comment:
@bdon This list is getting long, so long that Java complains and suggested I put it into a file instead. I put the full list into a file, and kept a shorter version here. This should be refactored to load from the file instead. Can you submit a patch for that, please?

Reply:
Let's defer this – it'd be better to switch to Natural Earth (at least for country labels).